lib/ffmpeg/qsv/vpp.py

```diff
@@ -63,6 +63,9 @@ def gen_vpp_opts(self):
           self.level, [0.0, 50.0, 100.0], procamp[self.vpp_op])
       elif self.vpp_op in ["sharpen"]:
         self.mlevel = mapsharp(self.level)
+      elif self.vpp_op in ["crop"]:
+        vpfilter.append("crop-left={left} crop-right={right}")
+        vpfilter.append("crop-top={top} crop-bottom={bottom}")
```

Comment:
This is incorrectly specified filter options.  Yet alone, it is also redundant and unnecessary because it needs to be handled by the dict entry below this.